modules/apps/commerce/commerce-frontend-js/src/main/resources/META-INF/resources/components/mini_cart/EditItem.js

```diff
@@ -41,15 +54,21 @@ const getProductOptionsURL = (channelId, productId) => {
 };
 
 function EditItem() {
+	const [cpInstance, setCPInstance] = useState({});
 	const [options, setOptions] = useState([]);
+	const [quantity, setQuantity] = useState(1);
+	const [quantitySelectorErrors, setQuantitySelectorErrors] = useState(false);
 	const [skuOptionsAtomState, setSkuOptionsAtomState] = useLiferayState(
 		skuOptionsAtom
 	);
-	const [cpInstance, setCPInstance] = useState({});
+	const [skuUnitOfMeasure, setSkuUnitOfMeasure] = useState(null);
```

Comment:
why here the default is null?

Reply:
It is null because the useState hook wants an initial value, but when we are declaring the variable we don't have any